docs/pages/admin-guides/management/guides/awsoidc-integration-rds.mdx

```diff
@@ -0,0 +1,289 @@
+---
+title: AWS RDS Enrollment Wizard
+description: Enroll AWS RDS databases with your Teleport cluster using an enrollment wizard.
+---
+
+The AWS RDS enrollment wizard is available in your Teleport cluster Web UI as
+part of the [AWS OIDC integration](./awsoidc-integration.mdx).
+
+The enrollment wizard configures AWS IAM resources and deploys a Teleport
+Database Service instance on ECS Fargate to proxy connections to AWS RDS
+databases.
+
+The enrollment wizard can be used to enroll RDS databases in a Teleport cluster.
+It can also be useful as an example serverless deployment of the Teleport database
+service on ECS.
+
```

Comment:
When I see "wizard", I think, "self-guided, in-product instructions", so seeing documentation for a wizard is a little counterintuitive to me. Is this a guide to setting up the wizard so, afterwards, the user and other users in the same organization can use it to enroll RDS databases?

Reply:
Same question. i feel it's more like a supplement to the guided wizard flow. links to this page will be added during the guided flow to have better explanation on how it works and provide some troubleshooting.

i wonder if the "architecture" section is a better place for how-it-works. but then troubleshooting will need a different place.

Reply:
> Is this a guide to setting up the wizard so, afterwards, the user and other users in the same organization can use it to enroll RDS databases?

No, it's purpose is:
1. explain how to find the RDS enrollment wizard and pre-reqs for using it
2. explain in detail exactly what the wizard is doing in AWS and why
3. provide thorough troubleshooting help

Other guided flows are also documented in this section, e.g https://goteleport.com/docs/admin-guides/management/external-audit-storage/

If you think it'd make more sense, I can drop the "wizard" terminology, but i'm not really sure what else to call it.